### DISCUSSION TITLE
MindSearch LLM

### DISCUSSION THREAD

**Discussion by onlyone-hyphen** [ItzCrazyKns/Vane#297] [Ideas]
https://huggingface.co/papers/2407.20183
https://github.com/InternLM/MindSearch

FYI, in case you missed it. This model released yesterday Apache 2.0, looks like a match made in heaven for Perplexica.

It beats Perplexities own model for "Deep Search"

**Comment by ItzCrazyKns** [comment]
Its not a model, its a competitor. The copilot feature will be better than Perplexity's copilot as well.

**Reply by onlyone-hyphen** [reply]
Oops lol, I should have spent more time reading it, I thought it was a fine-tune
